mindocr/utils/callbacks.py

```diff
@@ -274,14 +276,14 @@ def on_train_epoch_end(self, run_context):
             run_context (RunContext): Include some information of the model.
         """
         cb_params = run_context.original_args()
-        cur_epoch = cb_params.cur_epoch_num
+        cur_epoch = cb_params.cur_epoch_num + self.start_epoch
```

Comment:
fixed, by setting `initial_epoch=start_epoch` in `model.train`, `cb_params.cur_epoch_num` will automatically be update.

Reply:
My question was rather than loading weights as `ms.load_param_into_net(network, resume_param)` and then all additional values load manually, can we just load all values with `ms.load_param_into_net(cb_params.train_network, resume_param)` and avoid loading any additional values manually? Is this possible?

Reply:
There are many parameters that are not included in the `train_network`, such as `epoch_num`, ` loss_scale`, so they need to be saved and loaded additionally.